lua/lspconfig/configs/buflsp.lua

```diff
@@ -0,0 +1,21 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+
+    cmd = { 'buf', 'beta', 'lsp' },
+    filetypes = { 'proto' },
+    root_dir = function(fname)
+      return require('lspconfig.util').root_pattern('buf.work.yaml', '.git')(fname)
+    end,
```

Comment:
Looks like the lua linter failed with an unused var. You can also simplify root_dir since root_pattern returns a function.
```suggestion
return {
  default_config = {

    cmd = { 'buf', 'beta', 'lsp' },
    filetypes = { 'proto' },
    root_dir = require('lspconfig.util').root_pattern('buf.work.yaml', '.git'),
```